### DISCUSSION TITLE
Remove Player Collision

### DISCUSSION THREAD

**Discussion by DasLixou** [Minestom/Minestom#1069] [Help] [answered]
Hey! I wonder how I can turn off that player can push each other. In spigot it’s something like Teams but idk if this works with Minestom.

**Comment by HSGamer** [comment, marked as answer]
There is `Team` in Minestom, which you can create from `getTeamManager`
Here is the code I used to create that no-collison team
```java
var team = MinecraftServer.getTeamManager().createBuilder("any-team-name-here")
                .collisionRule(TeamsPacket.CollisionRule.NEVER)
                .build();
```
Now you can add the team to player with `player.setTeam(team)`.
